page-objects/pages/TASKListPage.js

```diff
@@ -20,6 +19,9 @@ const deleteTaskButtonInPopup = '.delete-modal button.btn-submit';
 const clickWithoutDueDate = "//*[text()='Task19']";
 const ungradedTask = "//div[text()='Task19']";
 const studentSubmitTask = "//td[text()='Boris']";
+const filterSelect = "//i[text() = 'add' and @class='material-icons']";
+const kurseSelect = "//div[contains(., 'Kurse...') and @class='md-list-item-content md-ripple']";
```

Comment:
Please use english wording here, coursesSelect instead of kurseSelect. Same for line 24 (kurseCheckbox) and line 80 (sortTasksKurse)